app/assets/javascripts/simulations.js

```diff
@@ -14,5 +14,18 @@ $(document).ready(function() {
       selectElement.id = selectElement.id.replace(/\d+/g, id);
     });
     tableBody.appendChild(rowCopy);
+    setRowsDeletable(true);
   });
+  setRowsDeletable(false);
 });
+
+setRowsDeletable = function(deletable) {
+  $('.simulation-delete-row').prop('disabled', !deletable); // ! because the boolean is for disabled, not enabled.
+};
+
+deleteRow = function(event) { // this gets called in the HTML by the onclick attribute
```

Comment:
I'm trying to lean away from coffeescript so I think you need to use `var` to define the deleteRow variable.